backend/middleware/authenticateToken.js

```diff
@@ -0,0 +1,31 @@
+const jwt = require("jsonwebtoken")
```

Comment:
This file is fine for now. One thing that would possibly add in is to check if the token that's being sent is from the correct user. Technically a malicious user could use an exposed JWT to potentially make requests on behalf of another user. (We can implement this later as it isn't the biggest concern right now)

Reply:
Additionally, if we do add this to the school's servers, it'll be protected by shibboleth (so quite a few security issues are resolved for us)